Rakefile

```diff
@@ -6,10 +6,7 @@ Bundler::GemHelper.install_tasks
 require "rake/testtask"
 task default: :test
 
-desc "Run tests"
-task :test do
-  Rake::TestTask.new do |t|
-    t.pattern = "spec/**/*_spec.rb"
-    t.verbose = true
-  end
+Rake::TestTask.new do |t|
+  t.pattern = "spec/**/*_spec.rb"
+  t.verbose = true
 end
```

Comment:
The `Rake::TestTask` already defines a task with a name `test` and the description "Run tests".